src/features/CourtCaseDetails/CourtCaseDetails.stories.tsx

```diff
@@ -37,17 +40,17 @@ export const DetailsNotLockedByAnotherUser: ComponentStory<typeof CourtCaseDetai
 
 DetailsNotLockedByAnotherUser.play = ({ canvasElement }) => {
   const canvas = within(canvasElement)
-  expect(canvas.getByText(courtCase.ptiurn)).toBeInTheDocument()
-  expect(canvas.getByText(courtCase.courtName)).toBeInTheDocument()
+  expect(canvas.getAllByText(courtCase.ptiurn)).toHaveLength(2)
```

Comment:
We could remove these fields from the old placeholder table so its only show in the summary box at top of the page:

![image](https://user-images.githubusercontent.com/22743709/229116812-654a9ba9-bb1b-46cd-8791-29e9b3820d82.png)

Reply:
All done!